x/mint/module_test.go

```diff
@@ -0,0 +1,29 @@
+package mint_test
+
+import (
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/x/mint"
+	"github.com/cosmos/cosmos-sdk/x/supply"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/tendermint/tendermint/abci/types"
+
+	"github.com/cosmos/cosmos-sdk/simapp"
+)
+
+func TestItCreatesModuleAccountOnInitBlock(t *testing.T) {
+	app := simapp.Setup(false)
+	ctx := app.BaseApp.NewContext(false, types.Header{})
+
+	app.InitChain(
+		types.RequestInitChain{
+			AppStateBytes: []byte("{}"),
+			ChainId:       "test-chain-id",
+		},
+	)
+
+	acc := app.AccountKeeper.GetAccount(ctx, supply.NewModuleAddress(mint.ModuleName))
```

Comment:
can you add a call before `InitChain` so that we test the module account is not set?

Reply:
right! Good one

Reply:
@fedekunze I have a problem with the Setup, it InitChain inside setup. And I cannot create a context before InitChain. Ideas?

Reply:
I am gonna try other option, as in staking.

Reply:
No, same issue, getting the context before init looks difficult.

Reply:
no worries then